app/gen/axios/findAgencies.ts

```diff
@@ -12,6 +12,7 @@ export async function findAgencies(
   const res = await client<FindAgenciesQueryResponse>({
     method: "get",
     url: `/agencies`,
+    baseURL: "https://zoning.planningdigital.com/api",
```

Comment:
Do you know why this `baseURL` property is getting set as a result of your changes? I would expect it to get set as a result of changes to `kubb.config.ts`, which doesn't show up in the diff.

Reply:
I do not.

Reply:
I regen'd and confirmed this doesn't seem to be due to any changes in this PR. My best guess is its related to [this commit upgrading Kubb](https://github.com/NYCPlanning/ae-cp-map/commit/dc97115b0b12512d0b4c4a9d606c418378405b35). I regen'd at a commit prior to that one and `baseURL` isn't getting set. I think what's happening is the newer version of Kubb is [pulling this url from the OAS spec file](https://github.com/NYCPlanning/ae-zoning-api/blob/main/openapi/openapi.yaml#L7). However, when we pass `baseURL` in our application code using the env var, that still takes precedence so this should be harmless. Resolving - cc @TangoYankee just so it's on your radar.